src/components/nav-bar.tsx

```diff
@@ -0,0 +1,142 @@
+import { Anchor, Box, Button, Flex, theme, useMinimumScreenSize } from '@cogito-study/alea';
+import React, { FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { Link } from 'gatsby';
+import { StyledGatsbyLink } from './styled/styled-gatsby-link';
+
+const pages: Record<string, string> = {
+  PRODUCT: '/',
+  BLOG: '/blog',
+  PRICING: '/pricing',
+  'ABOUT US': '/about',
+  CONTACT: '/contact',
+};
+
+const NavbarContainerLarge = styled(Flex)`
+  position: fixed;
+  top: 0;
+  width: 98%;
+
+  .active {
+    color: ${theme.colors.primary.normal};
+  }
+`;
+
+const NavbarContainerSmall = styled(Flex)`
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 2;
+`;
+
+const OverlayMenu = styled(Flex)`
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
+  width: 100vw;
+  height: 100vh;
+  z-index: -1;
+`;
+
+interface NavBarProps {
+  overlayMenuActive: boolean;
+  onMenuButtonClicked: () => void;
+}
+
+export const NavBar: FunctionComponent<NavBarProps> = ({ onMenuButtonClicked, overlayMenuActive }) => {
+  const large = useMinimumScreenSize('l');
+  return (
+    <>
+      {large && (
+        <NavbarContainerLarge
+          alignItems="center"
+          justifyContent="space-between"
+          border="1px solid"
```

Comment:
szerintem ide a sima `border={1}` eleg
```suggestion
          border={1}
```